MSAL/src/native_auth/controllers/reset_password/MSALNativeAuthResetPasswordController.swift

```diff
@@ -487,20 +487,20 @@ final class MSALNativeAuthResetPasswordController: MSALNativeAuthBaseController,
         username: String,
         pollInterval: Int,
         retriesRemaining: Int,
-        passwordResetToken: String,
+        continuationToken: String,
         event: MSIDTelemetryAPIEvent?,
         context: MSIDRequestContext
     ) async -> ResetPasswordSubmitPasswordControllerResponse {
         MSALLogger.log(level: .info, context: context, format: "Finished resetpassword/poll_completion")
 
         switch response {
-        case .success(let status, let continuationToken):
+        case .success(let status, let newContinuationToken):
```

Comment:
I think this might cause confusion in the future that this is just called `newContinuationToken` instead of something more specific. In other cases you use variable shadowing to hide the original continuation token so future developers might expect the new continuationToken to be just called continuationToken. Maybe instead we could consider creating a new variable called `oldContinuationToken` and leave this new continuation token named `continuationToken` so that we are consistent across the codebase.

Reply:
Do you suggest renaming the method parameter to "oldContinuationToken" at line 69? I'd prefer to rename all the new continuation token received from the API to "newContinuationToken"

Reply:
I think your suggestion is better alright - consistently rename all of the new continuation tokens to `newContinuationToken`. I think if we follow that pattern everywhere it will help make things clearer.